assets/tests/TaskSchemaValidationServiceSpec.js

```diff
@@ -44,17 +45,19 @@ describe('TaskSchemaValidationService', function() {
 
   describe('validateAllTasks', function() {
     it('should validate the structure of all sample tasks', function() {
+      var functions = Object.keys(Tsvs);
+      expect(functions.length).toEqual(NUM_FUNCTIONS);
```

Comment:
sure -- that works. I added some checks to make sure they're called each time for every task for every question, and messaging to let the person know if that didn't happen.